docs/configuration.md

```diff
@@ -446,7 +524,7 @@ The `redirect` subsection provides configuration for managing redirects from
 content backends. For backends that support it, redirecting is enabled by
 default. Certain deployment scenarios may prefer to route all data through the
 Registry, rather than redirecting to the backend. This may be more efficient
-when using a backend that is not colocated or when a registry instance is
+when using a backend that is not collocated or when a registry instance is
```

Comment:
["Colocated"](http://www.merriam-webster.com/dictionary/colocate) is the correct word. ["collocated"](http://www.merriam-webster.com/dictionary/collocate) has to due with word placement. "Co-located" may also be acceptable here.